### DISCUSSION TITLE
Newer version cannot open older version's Query Builder

### DISCUSSION THREAD

**Discussion by wellboy2022** [DaxStudio/DaxStudio#798] [General]
Latest version 2.17.3 cannot open older version's Query Builder. The query can open but not the Query Builder. It is not backward compatible.

**Comment by dgosbell** [comment]
Thanks for letting me know about this. I think I found what is causing this and figured out a fix which will be included in the next release.

If you wanted to try out a preview of the next release you could try the build here:
https://ci.appveyor.com/project/DarrenGosbell/daxstudio/builds/42556356/artifacts
